src/Module.js

```diff
@@ -41,6 +41,12 @@ class Module {
     this._transformCode = transformCode;
     this._depGraphHelpers = depGraphHelpers;
     this._options = options;
+
+    if (!this.moduleName && !this.isPolyfill() && !this.isAsset_DEPRECATED()) {
+      this.getName().then(name => {
+        this.moduleName = name;
```

Comment:
In other words, the `moduleName` property might or might not exists. I’m not really happy with this solution